mypy.ini

```diff
@@ -26,12 +26,45 @@ warn_unused_ignores      = True
 allow_untyped_defs = True
 check_untyped_defs = False
 
-[mypy-twisted.application.*]
+[mypy-twisted.application.app]
 allow_untyped_defs = True
 check_untyped_defs = False
 
-[mypy-twisted.application.runner.test.test_runner]
-allow_incomplete_defs = True
+[mypy-twisted.application.internet]
+allow_untyped_defs = True
+check_untyped_defs = False
+
+[mypy-twisted.application.reactors]
+allow_untyped_defs = True
+
+[mypy-twisted.application.service]
+allow_untyped_defs = True
+check_untyped_defs = False
+
+[mypy-twisted.application.strports]
+allow_untyped_defs = True
+check_untyped_defs = False
+
+[mypy-twisted.application.test.test_internet]
+allow_untyped_defs = True
+check_untyped_defs = False
+
+[mypy-twisted.application.test.test_service]
+allow_untyped_defs = True
+
+[mypy-twisted.application.twist._options]
+allow_untyped_defs = True
+
+[mypy-twisted.application.twist._twist]
+allow_untyped_defs = True
+
+[mypy-twisted.application.twist.test.test_options]
+allow_untyped_defs = True
+check_untyped_defs = False
+
+[mypy-twisted.application.twist.test.test_twist]
+allow_untyped_defs = True
+check_untyped_defs = False
```

Comment:
Longer list because instead of whole mode, we're excluding specific files, since we fixed some in the module.